<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerLogin</class>
 <widget class="QWidget" name="ManagerLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>677</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>677</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>677</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>管理员登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(243, 248, 254)
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 26pt &quot;Microsoft YaHei UI&quot;;
color:rgb(0, 157, 255);</string>
   </property>
   <property name="text">
    <string>管理员登录</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="M_PassWordLine">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background:rgb(255, 255, 255);
	color: black;
	border-radius:5px;
	font: 11pt &quot;微软雅黑&quot;;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>输入密码</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="ManagerNameLine">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	font: 11pt &quot;Microsoft YaHei UI&quot;;
	background:rgb(255, 255, 255);
	color: black;
	border-radius:5px;
	
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>输入管理员ID</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="LoginButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background:rgb(7, 148, 255);
	border-radius:5px;
	color: white;
	font: 18pt &quot;微软雅黑&quot;;
}</string>
   </property>
   <property name="text">
    <string>登      录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>440</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background:rgb(240, 240, 240);
	border-radius:5px;
	color: grey;
	font: 18pt &quot;微软雅黑&quot;;
}</string>
   </property>
   <property name="text">
    <string>取      消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>510</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 120, 212)</string>
   </property>
   <property name="text">
    <string>用户登录</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_ID">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_pass">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>CancelButton</tabstop>
  <tabstop>ManagerNameLine</tabstop>
  <tabstop>M_PassWordLine</tabstop>
  <tabstop>LoginButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
